cmd/insprd/memory/brokers/brokers.go

```diff
@@ -55,18 +60,24 @@ func (bmm *BrokerMemoryManager) Create(config brokers.BrokerConfiguration) error
 		return ierrors.NewError().Message("broker %s is not supported", broker).Build()
 	}
 
+	logger.Debug("subscribing broker to sidecar factory")
 	err = bmm.Factory().Subscribe(broker, factory)
-
 	if err != nil {
+		logger.Error("unable to subscribe broker")
 		return err
 	}
 
 	mem.Available[broker] = config
+	if mem.Default == "" {
+		mem.Default = broker
```

Comment:
You should call SetDefault so the transaction is registered on the log

Reply:
You're right